app/controllers/api/v2/moves_actions.rb

```diff
@@ -24,16 +24,14 @@ def create_and_render
     end
 
     def update_and_render
-      raise ActiveRecord::ReadOnlyRecord, "Can't change moves coming from Nomis" if move.from_nomis?
-
-      @move.assign_attributes(common_move_attributes)
-      action_name = @move.status_changed? ? 'update_status' : 'update'
-      @move.save!
-      @move.allocation&.refresh_status_and_moves_count!
+      move.assign_attributes(common_move_attributes)
+      action_name = move.status_changed? ? 'update_status' : 'update'
```

Comment:
`@move` is already memoized in the `move` method so this is just as fast as before, but cleaner (and resolves some strangeness where this method initially refers to `move` and then `@move`, which is what broke the specs originally).